modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -183,85 +202,90 @@
 /datum/job/cyborg
 	alt_titles = list(
 		"Cyborg",
-		"Robot",
 		"Android",
+		"Machine",
+		"Robot",
 	)
 
 /datum/job/detective
 	alt_titles = list(
 		"Detective",
+		"Forensic Scientist",
 		"Forensic Technician",
 		"Private Investigator",
-		"Forensic Scientist",
 	)
 
 /datum/job/doctor
 	alt_titles = list(
 		"Medical Doctor",
-		"Surgeon",
-		"Nurse",
 		"General Practitioner",
 		"Medical Resident",
+		"Nurse",
 		"Physician",
+		"Surgeon",
+		"Medical Student",
 	)
 
 /datum/job/engineering_guard //see orderly
 
 /datum/job/geneticist
 	alt_titles = list(
 		"Geneticist",
+		"Biologist",
```

Comment:
~~Is this not in use by science? It seems more like a Cytology-rename than genetics...~~
It's technically not in use, but it's really similar to the Xenobiologist. Also doesn't really cover the scope of the job, which works in genetics and DNA, not biology...
If you need a replacement idea you could use this one instead; simple combination of both the current ones
```suggestion
		"Genetics Researcher",
```

Reply:
I agree with Orion on this one.